docs/source/using/profiles.adoc

```diff
@@ -15,46 +15,52 @@ A profile is an instance of the Minishift VM along with all of its configuration
 The profile feature allows you to create and manage these isolated instances of Minishift.
 
 Each {project} profile is created with its own configuration (memory, CPU, disk size, add-ons, and so on) and completely independent of other profiles.
-See xref:../using/basic-usage.adoc#environment-variables[environment variables], if you want to apply some common configurations (CPU, memory etc) to all profiles.
+Refer to the use of xref:../using/basic-usage.adoc#environment-variables[environment variables], if you want to make sure that certain configuration settings, for example `cpus` or `memory`, get applied to all profiles.
 
-The current profile being used is called the active profile.
-Unless the global `--profile` flag is used, {project} commands are executed against the active profile.
-You can determine the active profile by using the xref:../command-ref/minishift_profile_list.adoc#[`minishift profile list`] command. 
-You can execute single commands against a non-active profile, by using the `--profile` flag.
-Example:
+There is always a so called active profile.
```

Comment:
"So called" has a negative connotation. In colloquial/conversational use, it carries the meaning of "says ______, but doesn't mean it." Let's avoid that connotation.

I think the original sentence ("the current profile being used is called the active profile") worked perfectly well.